orc8r/cloud/helm/orc8r/charts/certs/templates/admin-operator-certificate.yaml

```diff
@@ -0,0 +1,37 @@
+# Copyright 2021 The Magma Authors.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+{{- if .Values.create }}
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: {{ .Release.Name }}-admin-operator-certificate
```

Comment:
Minor nit: perhaps this is the right time to start calling these admin operator certs `default_admin_operator`. We support any number of admin operator certificates (i.e. certs giving admin permissions to the rest api), this is just the default one we create during provisioning